### PR TITLE
Workout Algorithm should now use all bodyparts

### PR DESCRIPTION
**Pull request by ThePhoneBeLinging**

### CODE REVIEW

File: src/main/java/Group15/Util/WorkoutAlgorithm.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ private static Exercise getValidExercise(List<BodyPart> selectedBodyParts, List<
             {
                 if (exercise.bodyParts.contains(bodyPart))
                 {
-                    continue;
+                    break;
```
Comment (WettyWednesday) [inline src/main/java/Group15/Util/WorkoutAlgorithm.java:95]: This seems to work, but i have had the issue that it just cycles back and forth between 2 muscles. To recreate i chose the first 4 muscles, with equipments: Barbell, Dumbbell, Cable and Machine. All i got was Bicep and Chest exercises, no Abdominal exercises or Calves.

**Review by WettyWednesday — CHANGES_REQUESTED** [review]
Im just requesting changes for if you can fix it. Otherwise we will just move forward

File: src/main/java/Group15/Util/WorkoutAlgorithm.java (hunk 1 above)
Comment (ThePhoneBeLinging) [reply src/main/java/Group15/Util/WorkoutAlgorithm.java:95]: Can't reproduce, so im marking it as resolved

Comment (WettyWednesday) [reply src/main/java/Group15/Util/WorkoutAlgorithm.java:95]: bro XD

**Review by emilsavino — APPROVED** [review]

**Review by WettyWednesday — APPROVED** [review]

**Pull request merged**